### PR TITLE
Added some updates (gold, Apigility, and HHVM)

### PR DESCRIPTION
**Pull request by adamculp**
Added images for Gold, Apigility, and HHVM, as well as info for Apigility.

### CODE REVIEW

File: source/_elephpants/2014-10-28-blue-apigility.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+sponsor: Zend Technologies
+reverse: Zend logo
+photos:
+    -
+        img: https://c2.staticflickr.com/2/1624/25011803152_3236e2c3be_o_d.jpg
+        link: https://www.flickr.com/photos/adamculp/25011803152/
+        caption: by adamculp
+    
+---
+A true rarity. This elephpant was to be given away to attendees at ZendCon 2014. However, leading up to ZendCon there were many rocket attacks over Israel. Someone felt the elePHPant looked like it was launching a rocket, and it was decided that the possibility was too sensitive at the time. So we created the elePHPant with the Z-Ray goggles instead. - only the prototype exists.
```
Comment (jrfnl) [inline source/_elephpants/2014-10-28-blue-apigility.md]: > So we created the elePHPant with the Z-Ray goggles instead

"So the elePHPant with the Z-Ray goggles was created instead"

**Comment by philipsharp** [comment]
@adamculp On Flickr these images are all listed as "All Rights Reserved". Is it okay to use them as CC BY-NC-SA?

**Comment by adamculp** [comment]
Oops, sorry about that. I will alter that on Flickr later today. #flying

**Comment by adamculp** [comment]
OK, updated the licensing. Thanks for pointing that out.

**Pull request merged**